tools/scripts/bumpenvs.yaml

```diff
@@ -1,33 +1,83 @@
+ap_northeast_1_agent_ami:
+  new: ami-08d8ce7792e9e4551
+ap_northeast_1_bastion_ami:
+  new: ami-02b658ac34935766f
+ap_northeast_1_master_ami:
+  new: ami-05d47ba59f0683b18
+ap_northeast_2_agent_ami:
+  new: ami-0657504cd814a3069
+ap_northeast_2_bastion_ami:
+  new: ami-064ab8637cf33f1bb
+ap_northeast_2_master_ami:
+  new: ami-0e077dbfdc14f6e35
+ap_southeast_1_agent_ami:
+  new: ami-0bbb1e46977bec0e4
+ap_southeast_1_bastion_ami:
+  new: ami-0c8e97a27be37adfd
+ap_southeast_1_master_ami:
+  new: ami-064a434d767f27a46
+ap_southeast_2_agent_ami:
+  new: ami-02fddef4d8bd31432
+ap_southeast_2_bastion_ami:
+  new: ami-099c1869f33464fde
+ap_southeast_2_master_ami:
+  new: ami-078e7a65a42dc74b5
+eu_central_1_agent_ami:
+  new: ami-0f12c86004d521a8b
+eu_central_1_bastion_ami:
+  new: ami-092391a11f8aa4b7b
+eu_central_1_master_ami:
+  new: ami-0e23b14b492db6c67
+eu_west_1_agent_ami:
+  new: ami-04ffea01cecb3a62d
+eu_west_1_bastion_ami:
+  new: ami-0823c236601fef765
+eu_west_1_master_ami:
+  new: ami-04bfff099c4b1d8ed
+eu_west_2_agent_ami:
+  new: ami-01b39b3103b0970f0
+eu_west_2_bastion_ami:
+  new: ami-09a1e275e350acf38
+eu_west_2_master_ami:
+  new: ami-0383d1a548b3a324c
 gcp_env:
-  new: pedl-environments-06318c7
-  old: pedl-environments-aaa3750
+  new: det-environments-7b8b1e9
+  old: det-environments-9b346b2
 tf1_cpu_hashed:
-  new: determinedai/environments:py-3.6.9-pytorch-1.4-tf-1.15-cpu-06318c7
-  old: determinedai/environments:py-3.6.9-pytorch-1.4-tf-1.15-cpu-aaa3750
+  new: determinedai/environments:py-3.6.9-pytorch-1.4-tf-1.15-cpu-7b8b1e9
+  old: determinedai/environments:py-3.6.9-pytorch-1.4-tf-1.15-cpu-9b346b2
 tf1_cpu_versioned:
   new: determinedai/environments:py-3.6.9-pytorch-1.4-tf-1.15-cpu-0.5.0
 tf1_gpu_hashed:
-  new: determinedai/environments:cuda-10.0-pytorch-1.4-tf-1.15-gpu-06318c7
-  old: determinedai/environments:cuda-10.0-pytorch-1.4-tf-1.15-gpu-aaa3750
+  new: determinedai/environments:cuda-10.0-pytorch-1.4-tf-1.15-gpu-7b8b1e9
+  old: determinedai/environments:cuda-10.0-pytorch-1.4-tf-1.15-gpu-9b346b2
 tf1_gpu_versioned:
   new: determinedai/environments:cuda-10.0-pytorch-1.4-tf-1.15-gpu-0.5.0
 tf2_cpu_hashed:
-  new: determinedai/environments:py-3.6.9-pytorch-1.4-tf-2.2-cpu-06318c7
-  old: determinedai/environments:py-3.6.9-pytorch-1.4-tf-2.2-cpu-aaa3750
+  new: determinedai/environments:py-3.6.9-pytorch-1.4-tf-2.2-cpu-7b8b1e9
+  old: determinedai/environments:py-3.6.9-pytorch-1.4-tf-2.2-cpu-9b346b2
 tf2_cpu_versioned:
   new: determinedai/environments:py-3.6.9-pytorch-1.4-tf-2.2-cpu-0.5.0
 tf2_gpu_hashed:
-  new: determinedai/environments:cuda-10.1-pytorch-1.4-tf-2.2-gpu-06318c7
-  old: determinedai/environments:cuda-10.1-pytorch-1.4-tf-2.2-gpu-aaa3750
+  new: determinedai/environments:cuda-10.1-pytorch-1.4-tf-2.2-gpu-7b8b1e9
+  old: determinedai/environments:cuda-10.1-pytorch-1.4-tf-2.2-gpu-9b346b2
```

Comment:
where is the `9b346b2` coming from?

Reply:
There are 3 different applications of bumpenvs in this commit.  You can see them in the individual commits.  There was one I missed from before, followed by the remove-PEDL update, followed by the multi-region update.

The reason I split up the last two was to purge the `pedl-` tags from the bumpenvs file.

Reply:
Side note: there's no meaningful difference in the actual contents of any of these changes, so if we cut a release without the most up-to-date AMIs it's not too big of a deal.